recipes/omniscidb-cpu/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "omniscidb-cpu" %}
+{% set version = "4.6.1" %}
+{% set sha256 = "04d69fd4b5d48e4bff937ad9c149c8a98746d54263485fcc3b840a4682bdda0d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}   # required for C++ headers
+    - cmake 3.12.*
+    - make
+    - maven
+    - openjdk 8.*
+    - wget
+    - clangdev 7.*
+    - go >=1.6
+    - bisonpp
+  host:
+    - snappy  # required for mapd-core > 4.5.0
+    - arrow-cpp 0.12.*
+    - blosc
+    - boost-cpp
+    - double-conversion
+    - libgdal 2.3.3
```

Comment:
The global pinning is `libgdal 2.4`, this will be incompatible with the rest of the stack. Unless you have a reason for breaking the global pinning I recommend to not pin and let conda-forge handle it.

Reply:
we are pinning here because there are some issues on omniscidb related to libgdal .. it fails with libgdal 3.0.0  .. if you want we can try pinning with 2.4 ... if it is not so bad .. this could be merged and we can try to remove the pinning when omniscidb has a new release.

Reply:
Ah, the central pinning is not applied on staged-recipes! You should not get `3.0` :-/

Ping to `2.4` for now.

Reply:
done! thanks @ocefpaf !